client/benches/tps/oneshot.rs

```diff
@@ -0,0 +1,11 @@
+//! Single trial of the benchmark
+
+mod lib;
+
+#[allow(clippy::expect_used)]
+fn main() {
+    let config = lib::Config::from_path("benches/tps/config.json").expect("Failed to configure");
+    let tps = config.measure().expect("Failed to measure");
+    iroha_logger::info!(?config);
```

Comment:
```suggestion
    iroha_logger::debug!(?config);
```

Reply:
I'd display the config by default for confirmation

Reply:
Another option is 
```rust
#[cfg(debug_assertions)]
iroha_logger::info!(?config);
#[cfg(not(debug_assertions))]
iroha_logger::debug!(?config);
```
If we're running the benchmark with `--nocapture` we might as well add `MAX_LOG_LEVEL=DEBUG` to see this kind of information.